README.md

```diff
@@ -65,6 +65,9 @@ The command line output will also include a localhost URL you can visit in your
 
 Make sure you don't already have a copy of the application running or the command will silently fail.
 
+You may need to run `pnpm graphql-codegen` in the `frontend` directory to update the code generated from the SMR API,
```

Comment:
Haven't pushed yet, but I added these two to the dev script. They were also in the postinstall script so they would have run with just `pnpm i` that `wails dev` should have ran unless using `-s` (skip wails building the frontend)

Reply:
I was not using the `-s` flag, and `wails dev` did not end up running them for me. Not sure if the cause of that was my old build directory or not.

Reply:
Looks like wails skips the `frontend:install` step if the `package.json.md5` it generates matches the `package.json` contents, even though it still takes a bit of time and prints `Installing frontend dependencies: Done.` rather than `Skipped` or something. https://wails.io/docs/guides/manual-builds/#install-frontend-dependencies